x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/translations.ts

```diff
@@ -532,7 +532,7 @@ export const IMPORT_RULE_BTN_TITLE = i18n.translate(
 export const SELECT_RULE = i18n.translate(
   'xpack.securitySolution.detectionEngine.components.importRuleModal.selectRuleDescription',
   {
-    defaultMessage: 'Select a Security rule (as exported from the Detection Engine view) to import',
+    defaultMessage: 'Select Security rules (as exported from the Detection Engine view) to import',
```

Comment:
With the new nav changes, not sure how accurate `as exported from the Detection Engine view` is. Perhaps `as exported from the Detection Rules page`?

<img width="1644" alt="Rules_-_Kibana" src="https://user-images.githubusercontent.com/2946766/126241388-03ac946e-8efa-4c38-b447-179ffb227101.png">

Reply:
makes sense to me